packages/sdk-react/src/components/counterparts/CounterpartDetails/CounterpartView/CounterpartView.tsx

```diff
@@ -104,6 +107,30 @@ export const CounterpartView = (props: CounterpartViewProps) => {
     deleteCounterpart(handleCloseDeleteCounterpartDialog);
   }, [deleteCounterpart, handleCloseDeleteCounterpartDialog]);
 
+  const { data: hasCounterpartDefault } =
```

Comment:
`const { data: contacts } =` ?

Reply:
We already had contacts constant in this file and scope. I will rename on contactsData